src/main/java/coms/w4156/moviewishlist/Services/MovieService.java

```diff
@@ -0,0 +1,15 @@
+package coms.w4156.moviewishlist.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import coms.w4156.moviewishlist.models.Movie;
+import coms.w4156.moviewishlist.repository.MovieRepository;
+
+@Service
+public class MovieService extends ServiceForRepository<Long, Movie, MovieRepository> {
```

Comment:
Instead of writing the same code over and over again for every model, I've created a fully generic abstract class for implementing services that wrap repositories.